src/loader/VRDataLoader.cpp

```diff
@@ -62,9 +56,10 @@ void VRDataLoader::load_txt_file(VRVolumeApp &vrVolumeApp, std::string &filename
           std::cerr << "at position " << vals[2] << " , " << vals[3] << " , " << vals[4] << std::endl;
           std::cerr << "text at position " << vals[2] << " , " << vals[3] << " , " << vals[5] << std::endl;
           std::cerr << "text Size " << vals[6] << std::endl;
-          std::cerr << "for Volume " << vals[7] << std::endl;
-          std::string label = p_filename.directoryPath() + OS_SLASH_LOCAL + vals[1];
-          vrVolumeApp.add_label(label, stof(vals[2]), stof(vals[3]), stof(vals[4]), stof(vals[5]), stof(vals[6]), stoi(vals[7]) - 1);
+          std::cerr << "text offset " << vals[7] << std::endl;
+          std::cerr << "for Volume " << vals[8] << std::endl;
```

Comment:
Should these be cerr? It seems like they're normal output that should use `std::cout`, no?

Reply:
No. std::cout doesnt log on release mode.

Reply:
Huh I never knew that - good to know!